JNG-3176 Add workflows and update xml files.

FarkasValentin opened the pull request.

FarkasValentin committed:
JNG-3176 Add workflows and update xml files.

FarkasValentin committed:
JNG-3176 Add skype message

kristofkakuszi committed:
JNG-3176: Moving from Wrecker to Github Actions.(karaf-features-eclipse-xtext)

robertcsakany commented on features/pom.xml:
```diff
@@ -45,102 +50,166 @@
-            <version>${eclipse-emf-karaf-features-version}</version>
-        </dependency>
-    </dependencies>
+
+    <profiles>
+        <profile>
+            <id>verify-feature</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.karaf.tooling</groupId>
+                        <artifactId>karaf-maven-plugin</artifactId>
+                        <extensions>true</extensions>
+                        <configuration>
+                            <enableGeneration>true</enableGeneration>
```
Wrong features are verified. These features belongs to apache-commons. The xtext features have to be validated are defines in features.xml.

kristofkakuszi committed:
JNG-3176: Moving from Wrecker to Github Actions.(karaf-features-eclipse-xtext)
Features/feature.xml updated.

kristofkakuszi committed:
JNG-3176: Moving from Wrecker to Github Actions.(karaf-features-eclipse-xtext)
Features/feature.xml updated, and other fixes in the file.

kristofkakuszi committed:
JNG-3176: Moving from Wrecker to Github Actions.(karaf-features-eclipse-xtext)
Features/feature.xml updated, and other fixes in the file.

kristofkakuszi committed:
JNG-3176: Moving from Wrecker to Github Actions.(karaf-features-eclipse-xtext)
Features/feature.xml updated, and other fixes in the file.

kristofkakuszi committed:
JNG-3176: Moving from Wrecker to Github Actions.(karaf-features-eclipse-xtext)
Features/feature.xml updated, and other fixes in the file.
Fixed missing descriptors and asm version in feature.xml
